project-maintainers.csv

```diff
@@ -383,9 +383,26 @@ Incubating,Thanos,Bartłomiej Płotka,Red Hat,bwplotka,https://github.com/thanos
 ,,Giedrius Statkevičius,,GiedriusS,
 ,,Povilas Versockas,,povilasv,
 ,,Dominic Green,Improbable,domgreen,
-Incubating,Flux,Hidde Beydals,Weaveworks,hiddeco,https://github.com/fluxcd/flux/blob/master/MAINTAINERS
+Incubating,Flux,Hidde Beydals,Weaveworks,hiddeco,https://github.com/fluxcd/community/blob/main/flux-project-maintainers.yaml
 ,,Michael Bridgen,Weaveworks,squaremo,
 ,,Stefan Prodan,Weaveworks,stefanprodan,
+,,Aurel Canciu, Sortlist,relu,
+,,Max Jonas Werner, D2iQ,makkes,
+,,Philip Laine, Xenit,phillebaba,
+,,Sunny, Weaveworks,darkowlzz,
+,,Alison Dowdney,,alisondy,
+,,Daniel Holbach, Weaveworks,dholbach,
+,,Scott Rigby, Weaveworks,scottrigby,
+,,Takeshi Yoneda, DMM.com,mathetake,
+,,Sean Eagan, AT&T,seaneagan,
+,,Somtochi Onyekwere, Weaveworks,SomtochiAma,
+,,Dinos Kousidis, Independent,dinosk,
```

Comment:
```suggestion
,,Dinos Kousidis, CrowdStrike,dinosk,
```

@dinosk - right?

Reply:
https://github.com/fluxcd/go-git-providers/blob/main/MAINTAINERS might need updating as well.

Reply:
https://github.com/dinosk confirms.

Reply:
Linkedin also matches 👍 https://www.linkedin.com/in/kousidis/

Will also update this corresponding MAINTAINERS file

Reply:
Yes, that's correct 👍